src/commands/projects/create/sql.ts

```diff
@@ -34,6 +35,13 @@ export default class SqlCommand extends AbstractProjectCreateCommand {
       exclusive: [],
       required: false,
     }),
+    language: flags.string({
+      char: 'l',
+      description: 'Choose the language you want to use for your project.',
+      options: Object.values(languages).map(language => language.name),
+      default: 'required',
+      required: false,
```

Comment:
[Question] What are these parameters used for ? I though this flag would have been required, accordingly to the prompts

Reply:
The default is used to have a value defined if the user doesn't pass the argument. It allows us to know if the language is requested for the prompts.
The required is set to false because otherwise the flag must be provided when typing the command (which makes the prompt useless).